### DISCUSSION TITLE
QT platform plugin "wayland" not found

### DISCUSSION THREAD

**Discussion by BNoordam** [bitcraze/discussions#1232] [General]
When running the WiFi video streaming example, i receive the following error message:
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in "/home/Bjorn/.local/lib/python3.11/site-packages/cv2/qt/plugins".
I tried several solutions online, such as at https://unix.stackexchange.com/questions/598099/could-not-find-the-qt-platform-plugin-wayland and https://stackoverflow.com/questions/69994530/qt-qpa-plugin-could-not-find-the-qt-platform-plugin-wayland, but to no avail. Could this error cause problems for my framerate? I am running opencv-python 4.5.5.64.

**Comment by gemenerik** [comment]
This warning does not affect your framerate.
